app/assets/stylesheets/common/base/sidebar.scss

```diff
@@ -143,6 +143,15 @@
   input.warning {
     border: 1px solid var(--danger);
   }
+  .checkbox-label {
```

Comment:
I would be surprised if we don't already have something like this for checkboxes, maybe ping Charlie and ask?